eslint.config.mjs

```diff
@@ -21,26 +21,6 @@ const config = createConfig([
     linterOptions: {
       reportUnusedDisableDirectives: 'error',
     },
-  },
-  {
-    rules: {
-      // Left disabled because various properties throughough this repo are snake_case because the
-      // names come from external sources or must comply with standards
```

Comment:
This doesn't apply to properties, just variables, so it should be easily avoided even when dealing with external sources.

There will still be occasions where we'll want to disable this for convenience reasons, but I don't think it needs to be turned off for the reason given here.